tfcoreml/_layers.py

```diff
@@ -1017,8 +989,8 @@ def argmax(op, context):
 
 def extract_image_patches(op, context):
   # use a big convolution layer (that has weights!) for this op
-  input_name = compat.as_bytes(op.inputs[0].name)
-  output_name = compat.as_bytes(op.outputs[0].name)
+  input_name = compat.as_str_any(op.inputs[0].name)
+  output_name = compat.as_str_any(op.outputs[0].name)
   ksizes = op.get_attr('ksizes')
   padding_type = op.get_attr('padding')
```

Comment:
This should also be changed to:
`padding_type = compat.as_str_any(op.get_attr('padding'))`